packages/client/wallets/smart-wallet/src/types/Error.ts

```diff
@@ -5,6 +5,10 @@ export const SmartWalletErrors = {
     CROSSMINT_SERVICE: "smart-wallet:crossmint-service.error",
     RUNNING_ON_SERVER: "smart-wallet:running-on-server",
     UNCATEGORIZED: "smart-wallet:uncategorized", // catch-all error code
+    ERROR_JWT_EXPIRED: "smart-wallet:jwt-expired",
+    ERROR_JWT_INVALID: "smart-wallet:jwt-invalid",
+    ERROR_JWT_DECRYPTION: "smart-wallet:jwt-decryption",
+    ERROR_JWT_IDENTIFIER: "smart-wallet:jwt-identifier",
```

Comment:
This isn't in the spec but seemed correct

Reply:
Other errors in the SDK seem to follow the pattern of:

`product:error-category.error` for exm `payments:payment-rejected.card-expired`

Maybe `smart-wallet:not-authorized.jwt-expired` or `smart-wallet:jwt-error.expired` would fit the pattern a little better